mfa/callbacks/about.md

```diff
@@ -0,0 +1,32 @@
+# Two-Factor Authentication Callbacks
+Two-Factor Authentication (2FA) Callbacks (also known as [webhooks](https://webhooks.pbworks.com/w/page/13385124/FrontPage)) rely on DLRs (Delivery Receipts) sent to the Two-Factor Authentication (2FA) platform by both Bandwidth's Domestic and International platform. Since these two platforms send different DLR formats, the 2FA platform standardizes them into a singular format for simplicity and ease of consumption. The 2FA platform then takes these standardized DLRs and sends them via an HTTP POST to the specified URL that was set in the `callbackUrl` field in the [initial request](../methods/code/messaging.md). The `callbackUrl` field can be any publicly addressable url.
+
+## Message Callbacks Concepts
+- Both Domestic and International callbacks have the same fields and have the same format. However, some of these fields differ in values to denote between the two systems.
+- Domestic callbacks are retried for 24 hours if the 2FA platform is unable to deliver it to you.
+- International callbacks are not retried if the 2FA platform is unable to deliver it to you.
+
+## Domestic Callbacks
+- The 2FA platform considers any message that is sent to a phone number in the United States and Canada to be domestic. This covers all phone numbers with the country code `+1`.
```

Comment:
The first sentence, with US and Canada only, contradicts the statement that follows about the +1 country code. Assuming we release this along with the changes in EDGE-955 I think we should be clear that we consider any message with a country code of 1 to be domestic.

Reply:
updated